pkg/loki/validation.go

```diff
@@ -23,6 +23,9 @@ func validateBackendAndLegacyReadMode(c *Config) []error {
 }
 
 func validateSchemaRequirements(c *Config) []error {
+	if len(c.SchemaConfig.Configs) == 0 {
+		return []error{}
+	}
```

Comment:
I think we should add a comment mentioning that the length of c.SchemaConfig.Configs is already validated in the `Validate()` function of the SchemaConfig itself.